src/fhir/1_Questionnaire-GDS.json

```diff
@@ -0,0 +1,745 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "CIRG-GDS",
+  "name": "GDS",
+  "code": [
+    {
+      "code": "48543-3",
+      "display": "Geriatric depression scale (GDS).short version panel"
+    }
+  ],
+  "title": "Geriatric depression scale (GDS).short version panel",
+  "status": "draft",
+  "meta": {
+    "profile": [
+      "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire|2.7"
+    ],
+    "tag": [
+      {
+        "code": "lformsVersion: 32.0.0"
+      }
+    ]
+  },
+  "item": [
+    {
+      "type": "choice",
+      "code": [
+        {
+          "code": "48512-8",
+          "display": "Are you basically satisfied with your life"
+        }
+      ],
+      "required": false,
+      "linkId": "/48512-8",
+      "text": "Are you basically satisfied with your life",
+      "answerOption": [
+        {
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-optionPrefix",
+              "valueString": "1"
```

Comment:
The scoring for this one needs to be updated, inconveniently it is not always 1 pt for yes, 0 points for no

Reply:
@achen2401 I am happy to go in and fix if you would like me to!

Reply:
@sierramatice The scoring is accounted for [here](https://github.com/uwcirg/asbi-screening-app/pull/35/files#diff-f79a90b783ca95682f6a089e00fb2a2284b12207f2f383b60b245034dc22b643)  line 187 to 295.  Will you check there to see if the score(s) are calculated as intended (some are given 1 points for yes, some are given 1 point for answer no)

Reply:
@achen2401 Looks fantastic, Sorry!